features/cli/cancel_unpaid_orders.feature

```diff
@@ -0,0 +1,49 @@
+@canceling_unpaid_orders
+Feature: Canceling unpaid orders
```

Comment:
The name of file should be `canceling_unpaid_orders.feature`

Reply:
https://github.com/Sylius/Sylius/pull/13683